bench/README.md

```diff
@@ -18,40 +28,214 @@ pip3 install -r bench/requirements.txt
 
 ## Preparation
 
+* Copy local accounts keys to folder `~/.dcl/keyring-test`:
+    ```bash
+    cp ./.localnet/node0/keyring-test/* ~/.dcl/keyring-test
+    ```
+* Enter to `bench` folder
+    ```bash
+    cd bench
+    ```
+* Build `docker-compose` file
+    ```bash
+    docker-compose build
+    ```
+* Open the docker-compose.yml:
+    * update fields `DCLD_VERSION`, `DCLD_NODE`, `DCLD_CHAIN_ID` for `args`.<br>
+        e.g.:
+        ```yml
+        DCLD_VERSION: v0.11.0
+        DCLD_NODE: tcp://host.docker.internal:26657
+        DCLD_CHAIN_ID: dclchain
```

Comment:
This should be done before the build step

Reply:
Yes, It's correct!